src/bentoml_cli/cloud.py

```diff
@@ -61,7 +61,8 @@ def login(shared_options: SharedOptions, endpoint: str, api_token: str) -> None:
         email=user.email,
     )
 
-    add_context(ctx)
+    new_config = add_context(ctx)
+    _ = new_config.set_current_context(ctx.name)
```

Comment:
Why do we update the current context? This would be a behavior change.

I often do `bentoml cloud login ... --context dev` and then `bentoml push --context dev`, because I want to use the dev environment for testing but don't want to update the current context.

Reply:
I see. Because if we have multiple context:

doing `bentoml cloud login ...` as copied from the UI doesn't actually log you in the context. As a user I would expect when i login my context will change to the one i just logged in.

Reply:
> doing `bentoml cloud login ...` as copied from the UI doesn't actually log you in the context.

Okay, but the approach to fixing it is wrong. We should instead, login using the **current active context** when no `--context` is given.

Reply:
I agree with @frostming here, we shouldn't updating the context automatically like this, which is confusing.

Probably better to be explicit at this case.